.github/workflows/build.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
         run: |
           sshpass -p "${{ secrets.PASSWORD }}" rsync -r -e "ssh -o StrictHostKeyChecking=no -p${{ secrets.PORT }}" ${{ secrets.USER }}@${{ secrets.HOST }}:ingenjorsarbeteforklimatet.se/content/data/ content/data/
```

Comment:
```suggestion
          sshpass -p "${{ secrets.PASSWORD }}" rsync -r -e "ssh -o StrictHostKeyChecking=no -p${{ secrets.PORT }}" ${{ secrets.USER }}@${{ secrets.HOST }}:greenhouse-majorna/content/data/ content/data/
```

Reply:
Put the image under our data store in `greenhouse-majorna/content/data/poster1.jpg`